components/file_converters/unstructured_fileconverter/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  unstructured-api:
+    image: "quay.io/unstructured-io/unstructured-api:latest"
+    ports:
+      - 8000:8000
+    restart: on-failure
+    healthcheck:
+        test: curl --fail http://localhost:8000/healthcheck || exit 1
+        interval: 10s
+        timeout: 1s
+        retries: 10
```

Comment:
do we need the compose file? I see in the readme we give instructions for plain `docker run ...`

Reply:
It is used to run the tests. I would prefer to update the readme.

Reply:
Why not using workflow `services`? See https://github.com/deepset-ai/haystack/blob/main/.github/workflows/tests.yml#L205

Reply:
done!